tfhe/examples/utilities/boolean_key_sizes.rs

```diff
@@ -42,7 +42,7 @@ fn client_server_key_sizes(results_file: &Path) {
         let test_name = format!("boolean_key_sizes_{params_name}_ksk");
 
         write_result(&mut file, &test_name, ksk_size);
-        write_to_json(
+        write_to_json::<u64, _>(
```

Comment:
u32 for boolean, doesn't the crypto param record already have the u32 set somewhere for Booleans ?